util/network-devp2p/src/host.rs

```diff
@@ -70,7 +71,9 @@ const SYS_TIMER: TimerToken = LAST_SESSION + 1;
 // for IDLE TimerToken
 const MAINTENANCE_TIMEOUT: Duration = Duration::from_secs(1);
 // for DISCOVERY_REFRESH TimerToken
-const DISCOVERY_REFRESH_TIMEOUT: Duration = Duration::from_secs(60);
+const DISCOVERY_REFRESH_TIMEOUT: Duration = Duration::from_secs(30);
```

Comment:
Not sure about the rationale for lowering this value.

Reply:
Now that we have the _fast_ one, I guess I can increase it.